app/models/user.rb

```diff
@@ -56,11 +56,11 @@ def mentor?
   end
 
   def npq?
-    npq_profiles.any?(&:active?)
+    npq_profiles.any?
```

Comment:
This is a change in behaviour (and not a good one...)

Reply:
Yup, corrected already! :)